plugins/zoom/icon_zoom/tasks/monitor_sign_in_out_activity/task.py

```diff
@@ -43,113 +44,133 @@ def run(self, params={}, state={}):
         output = [event.__dict__ for event in output]
         return output, new_state
```

Comment:
Is it necessary to use `has_more_pages` flag anyway? 😄

Reply:
> Is it necessary to use `has_more_pages` flag anyway? 😄

Not yet - that is going to be a future ticket for this plugin :)